File: webframework/src/main/java/com/brainz/framework/scheduling/task/SampleTask.java
package com.brainz.framework.scheduling.task;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Component;

import com.brainz.framework.sample.logging.controller.LoggingController;

@Component
public class SampleTask implements Runnable {
    private static Logger logger = LoggerFactory.getLogger(LoggingController.class);

    @Override
    public void run() {
        logger.info("Sample Task Execute!!");
    }

}

File: src/main/kotlin/co/brainz/framework/scheduling/task/SampleTask.kt
package co.brainz.framework.scheduling.task

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Component

@Component
public class SampleTask : Runnable {

	companion object {
		private val logger : Logger = LoggerFactory.getLogger(SampleTask::class.java)
	}
	
	override fun run() : Unit {
		logger.info("Sample Task Execute!!")
	}
}
